pkg/converter/convert_unix.go

```diff
@@ -58,17 +58,28 @@ func getBuilder(specifiedPath string) string {
 	return "nydus-image"
 }
 
-func getWorkdir(specifiedPath string) string {
-	if specifiedPath != "" {
-		return specifiedPath
+func ensureWorkDir(specifiedBasePath string) (string, error) {
+	var baseWorkDir string
+
+	if specifiedBasePath != "" {
+		baseWorkDir = specifiedBasePath
+	} else {
+		baseWorkDir = os.Getenv(envNydusWorkdir)
```

Comment:
Can we also name it as envNydusWorkDir like L62

Reply:
Fixed